setup.py

```diff
@@ -45,6 +45,7 @@ def findfiles(pat):
         'edx-opaque-keys==1.0.1',
         'pandas==0.24.2',
         'Unidecode==1.1.1',
+        'edx-ccx-keys==0.2.2',
```

Comment:
why is this necessary ?

Reply:
Opaque keys require it to recognize CCX keys, otherwise it will raise an InvalidKeyError. @andrey-canon